.github/workflows/ci.yml

```
@@ -79,14 +79,17 @@ jobs:
           check/mypy_.py
 
   coverage:
-    name: Pytest and Coverage check
+    name: Pytest and Coverage check for Python 3.7 - 3.11
```

Lets revert this. The actions will show it anyway.
```suggestion
    name: Pytest and Coverage check
```